app/src/main/java/com/omen273/crossLingo/MainActivity.kt

```diff
@@ -185,15 +202,16 @@ class MainActivity : AppCompatActivity() {
                     }
                     currentLayout.visibility = View.VISIBLE
                 }
-            }
 
-            for(i in currentRow.size until ITEMS_IN_ROW ) {
-                val layout = holder.tableRow.getChildAt(i) as LinearLayout
-                layout.visibility = View.INVISIBLE
+                for (i in currentRow.size until ITEMS_IN_ROW) {
+                    val layout = holder.tableRow.getChildAt(i) as LinearLayout
+                    layout.visibility = View.INVISIBLE
+                }
             }
         }
 
-        override fun getItemViewType(position: Int) = if(position == 0) 0 else 1
+        override fun getItemViewType(position: Int) =
+            if(position == 0) TOP_ROW else ONLY_CROSSWORDS_ROW
```

Comment:
Please rename ONLY_CROSSWORS_ROW to CROSSWORDS_ROW

Reply:
done